### ISSUE TITLE
API change ?

### ISSUE THREAD

**Issue by s-alad** [KinetixGaming/edpuzzle-cheat#2]
Did edpuzzle change the api so that "isCorrect" is always false?

**Comment by KinetixGaming** [comment]
No, what they did is they made it to where the variable isCorrect is only assigned a value once the Edpuzzle is complete
